Adyen/HttpClient/HttpClientWrapper.cs

```diff
@@ -65,7 +65,16 @@ public HttpRequestMessage GetHttpRequestMessage(string endpoint, string requestB
             httpRequestMessage.Headers.Add("ContentType", "application/json");
             httpRequestMessage.Headers.Add("Accept-Charset", "UTF-8");
             httpRequestMessage.Headers.Add("Cache-Control", "no-cache");
-            httpRequestMessage.Headers.Add("UserAgent", $"{_config.ApplicationName} {ClientConfig.UserAgentSuffix}{ClientConfig.LibVersion}");
+
+            if (!string.IsNullOrWhiteSpace(_config.ApplicationName))
+            {
+                httpRequestMessage.Headers.Add("UserAgent", $"{_config.ApplicationName} {ClientConfig.UserAgentSuffix}{ClientConfig.LibVersion}");
```

Comment:
The `ClientConfig.UserAgentSuffix` can have a better name (LibName?) and it shouldn't include `/`, I think this didn't get updated in this PR, but maybe it can be addressed.

Reply:
The `/` is here: https://github.com/Adyen/adyen-dotnet-api-library/blob/v32.0.0/Adyen/Constants/ClientConfig.cs#L14 and i don't want to change this value as it would be a `breaking-change`